src/gui/mainwindow.cpp

```diff
@@ -104,8 +109,38 @@ GUI::MainWindow::MainWindow(Pipeline::Pipeline *pipeline, QWidget *parent)
           SLOT(updateVideoFrame(cv::Mat)));
 }
 
+void GUI::MainWindow::updatePostureNotification(
+    PostureEstimating::PoseStatus poseStatus) {
```

Comment:
Would it be better if this just took the `PostureState` instead of the whole `PoseStatus`? I don't think we use any other field of the `PoseStatus` right?

Reply:
Yes good point, I have changed this now :)